src/neuroglancer/widget/segment_set_widget.ts

```diff
@@ -66,6 +66,43 @@ export class SegmentSetWidget extends RefCounted {
     this.registerDisposer(displayState.segmentColorHash.changed.add(() => {
       this.handleColorChanged();
     }));
+    this.registerDisposer(displayState.segmentSelectionState.changed.add(() => {
+      const segmentID = this.segmentSelectionState.selectedSegment.toString();
+      const segmentButton = <HTMLElement>document.querySelector(`[data-segID="${segmentID}"]`);
+      const existingHlight = document.querySelectorAll(`.segSelected`);
+      const white = vec3.fromValues(255, 255, 255);
+      const saturation = 0.5;
+      const removeClass_seg = (e: HTMLElement) => {
```

Comment:
[e.className.remove('segSelected'); ](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#Methods) does exactly what you want, I think.